features/step_definitions/auction_create_steps.rb

```diff
@@ -40,6 +40,10 @@
   end
 end
 
+Given(/^there is an auction with work in progress$/) do
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :delivered)
```

Comment:
wondering if we should have a `work_in_progress` trait since `delivered` feels kind of misleading here